src/components/InvoicesList/InvoicesList.js

```diff
@@ -24,6 +24,10 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
     maximumFractionDigits: 2,
   };
 
+  const dateFormatOptions = {
```

Comment:
What do you think about 

```suggestion
  const utdDateFormatOptions = {
```

Reply:
utd or utc?

Reply:
Sorry, UTC.

Reply:
updated the name to `utcDateFormatOptions`